package/scripts/precook_test.py

```diff
@@ -0,0 +1,237 @@
+            f.write("syscw: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("read_bytes: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("write_bytes: " + str(random.randint(0, char_lim)) + "\n")
+            f.write("cancelled_write_bytes: " + str(random.randint(0, char_lim)) + "\n")
+    else:
+        with open(io_fname, "w") as f:
+            f.write("rchar: " + str(fixed_value) + "\n")
+            f.write("wchar: " + str(fixed_value) + "\n")
+            f.write("syscr: " + str(fixed_value) + "\n")
+            f.write("syscw: " + str(fixed_value) + "\n")
+            f.write("read_bytes: " + str(fixed_value) + "\n")
+            f.write("write_bytes: " + str(fixed_value) + "\n")
+            f.write("cancelled_write_bytes: " + str(fixed_value) + "\n")
+
+
+def make_smaps(proc_pid, fixed_value, rand=False):
+    smaps_fname = os.path.join(proc_pid, "smaps")
+    if rand:
+        with open(smaps_fname, "w") as f:
+            mem_lim = 1000
+            f.write("Size: " + str(random.randint(0, mem_lim)) + " kB\n")
+            f.write("Pss: " + str(random.randint(0, mem_lim)) + " kB\n")
+            f.write("Rss: " + str(random.randint(0, mem_lim)) + " kB\n")
+            f.write("Swap: " + str(random.randint(0, mem_lim)) + " kB\n")
+    else:
+        with open(smaps_fname, "w") as f:
+            f.write("Size: " + str(fixed_value) + " kB\n")
+            f.write("Pss: " + str(fixed_value) + " kB\n")
+            f.write("Rss: " + str(fixed_value) + " kB\n")
+            f.write("Swap: " + str(fixed_value) + " kB\n")
+
+
+def make_net(proc_net, fixed_value, rand=False):
+    net_params = ["rx_bytes", "tx_bytes", "rx_packets", "tx_packets"]
+    for param in net_params:
+        net_fname = os.path.join(proc_net, param)
+        if rand:
+            with open(net_fname, "w") as f:
+                net_lim = 1000000
+                f.write(str(random.randint(0, net_lim)))
+        else:
+            with open(net_fname, "w") as f:
+                f.write(str(fixed_value))
+
+
+def make_nvidia(proc_nvidia, fixed_value):
+    # idx
+    smi_fname = os.path.join(proc_nvidia, "smi")
+    with open(smi_fname, "w") as f:
+        params = [
+            0,  # idx
+            pid,  # pid
+            "G",  # type
+            fixed_value,  # sm
+            fixed_value,  # mem
+            # enc, dec are not monitored metrics
+            0,  # enc
+            0,  # dec
+            fixed_value,  # fb
+            "python3",  # command
+        ]
+        for param in params:
+            f.write(str(param) + " ")
+
+
+def createPath(new_dir):
+    dir_name = os.path.join(os.getcwd(), "precooked_tests")
+    if not os.path.isdir(dir_name):
+        os.mkdir(dir_name)
+    dir_name = os.path.join(dir_name, new_dir)
+    if os.path.isdir(dir_name):
+        shutil.rmtree(dir_name)
+    os.mkdir(dir_name)
+    return dir_name
+
+
+def createTestMonotonous():
```

Comment:
I would call this `createTestMonotonic` - Monotonous has quite a different connotation!